tests/baselines/reference/api/typescript.d.ts

```diff
@@ -11220,5 +11220,7 @@ declare namespace ts {
      * @param compilerOptions Optional compiler options.
      */
     function transform<T extends Node>(source: T | T[], transformers: TransformerFactory<T>[], compilerOptions?: CompilerOptions): TransformationResult<T>;
+    function executeCommandLine(system: System, cb: ExecuteCommandLineCallbacks, commandLineArgs: readonly string[]): void;
+    type ExecuteCommandLineCallbacks = (program: Program | BuilderProgram | ParsedCommandLine) => void;
```

Comment:
Would prefer to keep this internal so that we dont have to maintain compatibility and stuff

Reply:
All of it? Or just the callback?

Reply:
All of it